package.json

```diff
@@ -45,8 +46,7 @@
     "config"
   ],
   "dependencies": {
-    "ember-cli-babel": "^5.1.6",
-    "ember-getowner-polyfill": "^1.0.0"
```

Comment:
In that case I believe we should solely use `ember-getowner-polyfill`, which added `ember-factory-for-polyfill` as a dependency in `1.2.0`. Will move things around. :v: